broker-core/src/test/java/io/zeebe/broker/workflow/message/MessageMappingTest.java

```diff
@@ -73,6 +73,16 @@
           .endEvent()
           .done();
 
+  private static final BpmnModelInstance NON_INT_BOUNDARY_EVENT_WORKFLOW =
```

Comment:
I think we don't need this test because it behaves like the interrupting boundary event. Also, I don't like that we need an extra parameter only for this case.